### DISCUSSION TITLE
Welcome to AIPlayground Discussions!

### DISCUSSION THREAD

**Discussion by Yonet** [aiadvocates/AIPlayground#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with you and other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Check out what we plan to build in [Projects tab](https://github.com/aiadvocates/AIPlayground/projects) and [issues tab](https://github.com/aiadvocates/AIPlayground/issues) and tell us what you think.
  * Check out the [AI Show Live Episodes](https://www.youtube.com/playlist?list=PLlrxD0HtieHinivDFATQd5qyjL6ijKHUF) you've missed and tell us you liked our dad jokes(or not 🥺)
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you are building.

**Comment by Yonet** [comment]
Hi!

**Comment by GithubJoseBr** [comment]
Hey  Ayşegül, will you talk about SciSharp today?

**Reply by Yonet** [reply]
Hi @GithubJoseBr, no, unfortunately. Looks like @sethjuarez or @cassieview might be able to one of these days!

**Comment by ghost** [comment]
Hi it is Rob from Canada!
I messed up my PATH when I was downgrading my python for a jupyter nb project. So I am going to be reinstalling everything... I was trying to figure it out if its better to do a vm or just to repurpose my old gaming laptop. That and hopefully some hiking in Jasper this weekend. Have a great day and thanks for the shows!

**Reply by sethjuarez** [reply]
Just saw this! What did you end up doing???
